kura/examples/org.eclipse.kura.example.eddystone.scanner/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Moreover, there are several copies of this file in our examples. Probably they should be deleted as well.